generate/templates/templates/nodegit.cc

```diff
@@ -1,7 +1,7 @@
 // This is a generated file, modify: generate/templates/nodegit.cc.
```

Comment:
That's there in case somebody is looking at the generated file `src/nodegit.cc` and is like "why can't I commit my changes.